frontends/web/src/routes/settings/settings.tsx

```diff
@@ -116,6 +118,23 @@ class Settings extends Component<Props, State> {
     this.setProxyConfig(proxy);
   };
 
+  private handleToggleSatsUnit = (event: React.SyntheticEvent) => {
+    const config = this.state.config;
+    if (!config) {
+      return;
+    }
+    const target = (event.target as HTMLInputElement);
+    var unit: BtcUnit = 'BTC';
+    if (target.checked) {
+      unit = 'SATS';
+    }
+
+    setConfig({
+      backend: { btcUnit: unit }
+    }).then(config => this.setState({ config }));
```

Comment:
shouldn't it be `etConfig({ frontend: { btcUnit: unit } })` instead of backend?

Reply:
not true, see e.g. setProxyConfig.

It's in the backend object if it is accessed by the backend.